### ISSUE TITLE
Docs typo for test_train_split?

### ISSUE THREAD

**Issue by karldw** [dask/dask-ml#428]
The [documentation for the `blockwise` argument](https://github.com/dask/dask-ml/blob/80785a9166b17bfde45f540c9f6a65eb558159e8/dask_ml/model_selection/_split.py#L379) of `train_test_split` says:

>     blockwise : bool, optional.
>         Whether to shuffle data only within blocks (True), or allow data to
>         be shuffled between blocks (False). Shuffling between blocks can
>         be much more expensive, especially in distributed environments.
>         The default behavior depends on the types in arrays. For Dask Arrays,
>         the default is True (data are not shuffled between blocks). For Dask
>         DataFrames, the default and only allowed value is True (data are
>         shuffled between blocks).

Should that actually read "For Dask DataFrames, the default and only allowed value is True (data are **not** shuffled between blocks)."?

```

**Comment by TomAugspurger** [comment]
It's a typo. For dask DataFrame, the default is `False` (so data *are* shuffled between blocks, via DataFrame.random_split).
